app/models/project.rb

```diff
@@ -72,6 +72,18 @@ def repository_homepage
     "//#{Rails.application.config.samson.github.web_url}/#{github_repo}"
   end
 
+  def repository_homepage_gitlab
+    "//#{ENV['GITLAB_URL']}/#{github_repo}"
```

Comment:
Would be better if `github_repo` was only used for Github or rename it to a more generic name. Also, mind putting the gitlab url as a config var like [this](https://github.com/zendesk/samson/blob/master/config/application.rb#L57)?